.github/workflows/pr-manual-qa-plan.yml

```
@@ -0,0 +1,247 @@
+  execute:
+    needs: [resolve, automation]
+    if: >-
+      needs.automation.outputs.automation_passed == 'true' &&
+      needs.resolve.outputs.needs_execution == 'true'
+    permissions:
+      contents: read
+      pull-requests: write
+      statuses: write
+    uses: ./.github/workflows/pr-manual-qa-execute.yml
+    with:
+      pr_number: ${{ needs.resolve.outputs.pr_number }}
```

actionlint (1.7.12)

[error] input "pr_number" is typed as number by reusable workflow "./.github/workflows/pr-manual-qa-execute.yml". string value cannot be assigned

(expression)